navigation.toml

```
@@ -18,6 +18,7 @@ footer_internal = [
     { title = "Legal", href = "/legal" },
     { title = "Law Enforcement Guidelines", href = "/legal/law-enforcement-guidelines" },
     { title = "Contact", href = "/contact" },
+    { title = "Status", href = "https://status.matrix.org" },
```

To make it clear this isn't status for the whole federation:
```suggestion
    { title = "Homeserver Status", href = "https://status.matrix.org" },
```

---

But it is also *not only* the status for only the homeserver (see ticket)